src/core/utils/addons.js

```diff
@@ -25,6 +26,8 @@ export const getErrorMessage = ({
   invariant(i18n, 'i18n is required');
 
   switch (error) {
+    case ERROR_CORRUPT_FILE:
+      return i18n.gettext('Error occurred due to corrupt file');
```

Comment:
Does it seems to be good error message or do you have any better suggestion?

Reply:
Mmh, maybe we should have something similar to [this](https://searchfox.org/mozilla-central/rev/116bd975c30746ddefc3d20e6947d1871469354f/browser/locales/en-US/chrome/browser/browser.properties#231):

> Installation aborted because the add-on is unverified.